File: examples/function/list/node/listFunctions.js
const { FunctionService } = require("m3o/function");

// List all the deployed functions
async function listFunctions() {
  let functionService = new FunctionService(process.env.MICRO_API_TOKEN);
  let rsp = await functionService.list({});
  console.log(rsp);
}

listFunctions();

File: examples/function/list/go/listFunctions.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/function"
)

// List all the deployed functions
func ListFunctions() {
	functionService := function.NewFunctionService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := functionService.List(&function.ListRequest{})
	fmt.Println(rsp, err)
}
